lamin_cli/_snapshot.py

```diff
@@ -0,0 +1,80 @@
+from __future__ import annotations
+
+import os
+import shutil
+import subprocess
+import sys
+
+import lamindb_setup as ln_setup
+
+if os.environ.get("NO_RICH"):
+    import click as click
+else:
+    import rich_click as click
+
+
+@click.group()
+def snapshot():
+    """Manage snapshots."""
+
+
+@snapshot.command("create")
+@click.option(
+    "--upload/--no-upload",
+    is_flag=True,
+    help="Whether to upload the snapshot.",
+    default=True,
+)
+def create(upload: bool):
+    """Create a SQLite snapshot of the current instance."""
+    if not ln_setup.settings._instance_exists:
+        raise click.ClickException(
+            "Not connected to an instance. Please run: lamin connect account/name"
+        )
+
+    instance_owner = ln_setup.settings.instance.owner
+    instance_name = ln_setup.settings.instance.name
+    export_dir = f"{instance_name}_export"
+
+    ln_setup.connect(f"{instance_owner}/{instance_name}", use_root_db_user=True)
+
+    modules_without_lamindb = ln_setup.settings.instance.modules
+    modules_complete = modules_without_lamindb.copy()
+    modules_complete.add("lamindb")
+
+    import lamindb as ln
+
+    ln.settings.verbosity = "error"
+
+    ln.track()
+    ln_setup.io.export_db(module_names=modules_complete, output_dir=export_dir)
+    ln.finish()
+
+    import_code = f"""
```

Comment:
Why not make this a proper module that you run as a script? That'd be much easier to maintain than a triple string quote? 🤔 

`create_sqlite_clone_and_import_db.py`

Or something like this?

Reply:
Thank! I will deal with this flaw of the tracking design. The tracking design will be better in addressing this and a few other design issue.